packages/documentation/src/pages/forms/index.mdx

```diff
@@ -13,16 +13,24 @@ graph LR
     C -- Submission --> D(Confirmation Page)
 ```
 
-## When to use the Forms System
+VAFS consists of several different pieces:
 
-The Forms system is best suited for forms that
+- Widgets and fields for common types of data collected on forms
+- Common validation functions used for different types of data
+- A form data flow process built on top of the React JSON Schema Form library
```

Comment:
I'm hesitant to link to the library more than we already are, since we'll probably just fold the code into our codebase. Plus, we're on a fork.

I struggled a little with this line. I want to convey that it provides a flow for how form data moves through an application, so you don't have to write your own way of taking user input, running validation, storing the data, and re-updating the UI with any conditionally displayed changes.